packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.Props.ts

```diff
@@ -54,9 +54,11 @@ export interface IComboBoxProps extends ISelectableDroppableTextProps<IComboBox>
 
   /**
    * Whether the ComboBox auto completes. As the user is inputing text, it will be suggested potential matches from the list of options. If
-   * the combo box is expanded, this will also scroll to the suggested option, and give it a selected style. Defaults to false.
+   * the combo box is expanded, this will also scroll to the suggested option, and give it a selected style.
+   *
+   * @default "on"
    */
-  autoComplete?: string;
+  autoComplete?: 'on' | 'off';

```

Comment:
This is a breaking change although it seems pretty safe since anything else before wouldn't have been handled (but will now be broken). @dzearing can make the call on if this should go the deprecated route or not.

Reply:
It's also DEFINITELY a breaking change in the sense that if user code was something like

```
let autoCompleteConfig: string = 'on';

let comboBoxProps: IComboBoxProps = {
  // ...
  autoComplete: autoCompleteConfig
};
```

The fairly reasonable code above which used to work, will now no longer work (it will throw a Type error).

Reply:
I didn't intend to create a breaking change.  I'm a bit unfamiliar with this codebase and typescript (this is my first PR to Fabric).  I'm happy keeping it as `string` and mentioning the available options in the comment, if that's what the maintainers would prefer.  I was just trying to make the docs as clear as possible.

Reply:
No worries at all Ian. That was meant to be informational and not accusatory :)

Reply:
I actually really like your change - we just need to be careful about the side effects of the change if we were to take it.

Reply:
Honestly I think it's reasonable. It technically is only a breaking change if you're explicit about the type of the variable passed in. But I think 99% of usage scenarios would just be inlining 'on' or 'off' in the prop.

I really hate this prop honestly. It's consistent with the html prop but it really should have been a true/false in html land.

I vote we change it to `'on' | 'off'` and deal with it if there is a rare case it breaks a partner. They will have a very clear error message and it will be very obvious about how to fix.